python/cog/server/webhook.py

```diff
@@ -2,12 +2,15 @@
 from typing import Any, Callable, Set
 
 import requests
+import structlog
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.util.retry import Retry  # type: ignore
 
 from ..schema import Status, WebhookEvent
 from .response_throttler import ResponseThrottler
 
+log = structlog.get_logger("cog.server.webhook")
```

Comment:
Not a big deal, but in files other than the entrypoint you can get away with `...get_logger(__name__)`.